### PR TITLE
Events box on homepage

### PR DESCRIPTION
**Pull request by ChenProm-USDA**
Show dates on the next line.
Layout is controlled by _theme/jekyll-theme-legumeinfo/_includes/card-items.html.
To fix:
Add a copy of this file in the root _includes folder and update with code below.

```<li><a href="{{ include.url }}"><b>{{ include.title }}</b></a>{% if include.meta %} <div>{{ include.meta }}</div>{% endif %}</li>```

Screenshot 2024-09-11 at 16 16 02

### CODE REVIEW

File: _includes/card-item.html
Comment (nathanweeks) [inline _includes/card-item.html:1]: Is this generally visually helpful? If so, might be beneficial to add to the upstream theme?
@jd-campbell

**Comment by maxglycine** [comment]
Chen, remember the contrast of the text matters.  You need to be sure all the NON-link text is black (#000000). It does not look black to me.

File: _includes/card-item.html
Comment (jd-campbell) [reply _includes/card-item.html:1]: @nathanweeks What do you mean by upstream theme? Are you referring to the LIS theme?

Comment (jd-campbell) [reply _includes/card-item.html:1]: Chen, 
Thank you this looks great. I appreciate that you mentioned what you changed, no I can put in my website management notes.

**Review by jd-campbell — APPROVED** [review]
I approve. I will not merge until @nathanweeks replies to my question regarding 'upstream theme'.

**Comment by ChenProm-USDA** [comment]
@jd-campbell should I change the text color as mention by @maxglycine before the merge?

**Comment by maxglycine** [comment]
Yes, you should change it before merge.  Text contrast is a section 508 issue and we need to keep main consistent.

**Comment by ChenProm-USDA** [comment]
@jd-campbell Text color has been fixed and changes pushed. 
Screenshot 2024-09-12 at 09 56 45

**Pull request merged**
